src/Stratis.Bitcoin/Consensus/ConsensusManager.cs

```diff
@@ -685,7 +685,7 @@ private async Task<ConnectBlocksResult> ReconnectOldChainAsync(ChainedHeader cur
 
             if (connectBlockResult.Succeeded)
             {
-                var result = new ConnectBlocksResult(false, false);
```

Comment:
@Aprogiena @noescape00 can you guys comment here, I think the false here is a bug? I changed it true?

Reply:
Why do you think it is a bug? It looks correct as it was. Notice who consumes that result

Reply:
This `if` is stating that the result succeeded. So why would we return a result with `Suceeded` being false?

Reply:
Because those are two different operations. The operation for which we want to return the result failed.

Reply:
👍